collections/Selden_supra/MS_Selden_Supra_30.xml

```diff
@@ -38,43 +37,79 @@
                   <decoDesc>
-                     <decoNote type="decInit">Decoration<p>Good initial (defaced). (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 2)</p>
-                     </decoNote>
+                     <decoNote type="decInit">Large opening initial decorated with knots (heavily worn), with enlarged letters on the first line coloured in red, blue, and yellow. (<ref target="https://catalog.hathitrust.org/Record/000468709">Pächt and Alexander</ref> iii. 2) </decoNote>
                   </decoDesc>
+                  <bindingDesc>
+                     <binding notAfter="1700" notBefore="1600">
+                        <p>Limp parchment cover, 17th century.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
                <history>
                   <origin>
                      <origDate calendar="Gregorian" notAfter="0750" notBefore="0700">8th century, first half</origDate>
-                     <origPlace cert="low">
-                        <country key="place_7002445">English</country>, <region>South</region>, <settlement key="place_1009430">Thanet</settlement> (?)<!--ORIGINAL: English, South - Thanet (?)--></origPlace>
+                     <origPlace><country key="place_7002445">English</country>, <region>south</region></origPlace>
                   </origin>
-                  <provenance type="MLGB3_provenance_evidence"
-                              source="http://mlgb3.bodleian.ox.ac.uk/mlgb/book/1612">
-                     <orgName role="fmo" key="org_165174766">Canterbury, Kent, Benedictine abbey of St Augustine (originally of St Peter and St Paul)</orgName>: "Di. I. G. III". (<ref target="http://mlgb3.bodleian.ox.ac.uk/mlgb/book/1612">MLGB3</ref>: inferred evidence). </provenance>
-                  <provenance notAfter="1654" notBefore="1584" resp="#MMM">
-                     <persName key="person_51728429" role="fmo">John Selden, 1584–1654</persName>
-                  </provenance>
-                  <acquisition resp="#MMM" when="1659">Acquired by the Bodleian in 1659</acquisition>
+                  <provenance><settlement key="place_1009430">Thanet</settlement>, Kent, <orgName key="org_211802270">Minister Abbey</orgName>: the name <persName key="person_80945311">Eadburg</persName> is inscribed on pp. 2 and 47 in dry point, probably referring to the woman documented as abbess from at least 733 until her death in 748×61. The female pronouns in the added prayers on p. 70 indicate continued ownership at Minster-in-Thanet or in another community of nuns. See Jessica Hodgkinson and John Barrett, <q><ref target="https://blogs.bodleian.ox.ac.uk/theconveyor/women-in-the-margins-eadburg-and-bodleian-library-ms-selden-supra-30/">Women in the margins: Eadburg and Bodleian Library, MS. Selden Supra 30</ref></q> (November 2022).</provenance>
```

Comment:
could "probably" be too strong? and could it be worth mentioning the connection between Minster and St Augustine's via the translation of Mildrith's relics?